UE/Application/Ports/UserPort.cpp

```diff
@@ -117,8 +117,12 @@ void UserPort::showTalking()
 
```

Comment:
We should also clear the textbox where's all messages appered, but i couldn't find a function to do so. We should ask about it in next technical issues.

Reply:
Yeah, I couldn't find function for that. I think we must leave it as it is.